stdparams.go

```diff
@@ -0,0 +1,36 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+// ParamsKey is the request context key under which URL params are stored.
+//
+// This is only present from go 1.7.
+const ParamsKey = "_httprouter_params"
```

Comment:
It's even better to use a type, rather then a string. For example like in gRPC: https://github.com/grpc/grpc-go/blob/9e3a674ceba65708273cf1cd8e71a1bdce68107b/peer/peer.go#L54-L65.